tutorials/preprocessing/35_artifact_correction_regression.py

```diff
@@ -23,146 +24,165 @@
 Other approaches like :ref:`ICA <tut-artifact-ica>` or
 :ref:`SSP <tut-artifact-ssp>` will likely work better for ECG.
 
+Furthermore, regression approaches are usually performed in situations where
+there are few channels available, and removing an entire signal component is
+undesirable. Hence, most articles on the topic concern EEG and it is
+unusual to see the technique applied to MEG. For this reason, we will restrict
+the analysis in this tutorial to EEG data only.
+
+
 Prepare the data
 ^^^^^^^^^^^^^^^^
 
 We begin as always by importing the necessary Python modules and loading some
-data. In this case we use data from :ref:`Brainstorm <tut-brainstorm-auditory>`
-because it has some clear, large blink artifacts. We then crop it to 60
-seconds, set some channel information, then process the auditory evoked data.
-
-Note that there are other corrections that are useful for this dataset that we
-will not apply here (see :ref:`tut-brainstorm-auditory` for more information).
+data. The MNE-Sample dataset has some clear, large blink artifacts, especially
```

Comment:
MNE-Sample dataset could be linked to https://mne.tools/dev/overview/datasets_index.html#sample